radar-spring-auth/src/main/kotlin/radar/spring/auth/common/Authorization.kt

```diff
@@ -19,12 +20,13 @@ interface Authorization<T> {
         user: String? = null,
         source: String? = null
     ): Boolean {
-        return hasPermission(token, permission, entity, permissionOn, project, user, source) &&
-            hasRole(token, project, role) &&
-            hasScopes(token, scopes) &&
-            hasAuthorities(token, authorities) &&
```

Comment:
Ah the authorities list was removed from the `RadarToken` object, so I thought it was removed, but it can be taken from the roles. I've added it back now.